clustered/common/src/main/java/org/ehcache/clustered/common/internal/messages/EhcacheMessageType.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright Terracotta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.ehcache.clustered.common.internal.messages;
+
+import org.terracotta.runnel.EnumMapping;
+
+import java.util.EnumSet;
+
+import static java.util.EnumSet.of;
+import static org.terracotta.runnel.EnumMappingBuilder.newEnumMappingBuilder;
+
+/**
+ * EhcacheMessageType
+ *
+ * Whenever you edit this, you must think about enum mapping and helper methods
+ */
+public enum EhcacheMessageType {
+  // Lifecycle messages
+  CONFIGURE,
```

Comment:
Would it be possible to reuse these enums as operation enums in messages ? Right now we end up maintaining same set of enums at multiple places.